packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/DynamicTable/DefaultButton/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Link } from '@strapi/helper-plugin';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+const MESSAGES_MAP = {
```

Comment:
I'm not sure if I think this is a good pattern. Why? I fear that if the code which is calling these is modified or deleted these could be forgotten. What do you think about inlining it and make it a conditional? Or do you think you will add more keys?

Reply:
Hey, sorry I need some more infos, what do you mean? the DefaultButton component itself or the buttonType category?

Reply:
Nevermind: I thought about it again and I think your idea works great :)